.github/workflows/release_sdk.yml

```diff
@@ -0,0 +1,38 @@
+name: Release SDK
+
+on:
+  push:
+    tags:
+      - 'sdk_*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+      - name: Install Java and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install GNUPG2 & Import Key
+        env:
+          GPG_KEY_PRIVATE: ${{ secrets.GPG_PRIVATE_KEY }}
+        run: |
+          sudo apt-get install gnupg2 -y
```

Comment:
If you want to keep this clean you can make a script file, but is just a nitpick

Reply:
yeah we can!! now i just want to try to release 😄 i can open another PR afterwards